tmp_script.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# Rung by passing in the function name to call as first parameter to the script.
+# E.g.  ./tmp_script.sh build_rust
+#
+# Be aware that this script will in a follow up PR be replaced with an ob runner.
+
+set -eux
+cd "$(dirname $0)"
+
+function install_async_bindgen_cli() {
+    cargo install \
+        --git https://github.com/xaynetwork/xayn_async_bindgen.git \
+        async-bindgen-gen-dart \
+        "$@"
+}
+
+function gen_dart_genesis_extension() {
+    cd discovery_engine_core
+    # Doesn't respect cargo config option, but good enough for us for now
+    "${CARGO_INSTALL_ROOT:-${CARGO_HOME:-$HOME/.cargo}}/bin/async-bindgen-gen-dart" \
```

Comment:
@janpetschexain @Robert-Steiner @biernacki can you test that this works on mac?

Reply:
Maybe we should use a custom install root in the workspace (git-ignored)?

Then we also don't have a problem with any situation where the install root is 
changed in a cargo config file (`.cargo/config.toml`).

Reply:
Seems a good idea

Reply:
Ok I reposted the comment on #61 so that's not lost.